stdlib/public/SDK/Foundation/URLRequest.swift

```diff
@@ -241,31 +241,28 @@ public struct URLRequest : ReferenceConvertible, Equatable, Hashable {
 extension URLRequest : CustomStringConvertible, CustomDebugStringConvertible, CustomReflectable {
 
     public var description: String {
-        if let u = url {
-            return u.description
-        } else {
-            return "url: nil"
-        }
+      return url?.description ?? "url: nil"
     }
     
     public var debugDescription: String {
         return self.description
     }
     
     public var customMirror: Mirror {
-        var c: [(label: String?, value: Any)] = []
-        c.append((label: "url", value: url))
-        c.append((label: "cachePolicy", value: cachePolicy.rawValue))
-        c.append((label: "timeoutInterval", value: timeoutInterval))
-        c.append((label: "mainDocumentURL", value: mainDocumentURL))
-        c.append((label: "networkServiceType", value: networkServiceType))
-        c.append((label: "allowsCellularAccess", value: allowsCellularAccess))
-        c.append((label: "httpMethod", value: httpMethod))
-        c.append((label: "allHTTPHeaderFields", value: allHTTPHeaderFields))
-        c.append((label: "httpBody", value: httpBody))
-        c.append((label: "httpBodyStream", value: httpBodyStream))
-        c.append((label: "httpShouldHandleCookies", value: httpShouldHandleCookies))
-        c.append((label: "httpShouldUsePipelining", value: httpShouldUsePipelining))
+        let c: [(label: String?, value: Any)] = [
+          ("url", url as Any),
+          ("cachePolicy", cachePolicy.rawValue),
+          ("timeoutInterval", timeoutInterval),
+          ("mainDocumentURL", mainDocumentURL as Any),
+          ("networkServiceType", networkServiceType),
+          ("allowsCellularAccess", allowsCellularAccess),
+          ("httpMethod", httpMethod as Any),
+          ("allHTTPHeaderFields", allHTTPHeaderFields as Any),
+          ("httpBody", httpBody as Any),
+          ("httpBodyStream", httpBodyStream as Any),
+          ("httpShouldHandleCookies", httpShouldHandleCookies),
+          ("httpShouldUsePipelining", httpShouldUsePipelining),
+        ]
```

Comment:
For a while, something like this would be too complex for the type checker, hence (I'm guessing) the `c.append(...)` statements before. Have you checked to make sure that this is not still the case?